src/main/resources/swaggerDocs/swagger.yaml

```diff
@@ -879,10 +879,22 @@ paths:
           format: JSON
         - name: path
           in: query
-          description: SPARQL property path composed of full IRIS. This is used to connect the data resource to count (RDF graph world) to the ontology world. E.g. `<http://purl.org/has_state>/<http://purl.org/describes_phenotype>`
+          description: SPARQL property path composed of full IRIS. This is used to connect the data resource to count (RDF graph world) to the ontology world. E.g. `<http://purl.obolibrary.org/obo/RO_0002158>/<http://purl.obolibrary.org/obo/RO_0002487>`
```

Comment:
Why not use a property path that's relevant for our semantic similarity queries? `has_state/describes_phenotype` would be good, with the correct IRIs.